run_integration.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
I don't think this file's name is accurate.  `run_integration` sounds like it could be running like a Slack integration or something similar.

Should we call it `run_integration_tests.py` ?

Reply:
removed.